### PR TITLE
Added chrome driver

### PR DESCRIPTION
**Pull request by adorin**
I documented the Chrome driver, as suggested by @aik099 in https://github.com/minkphp/Mink/pull/738.

### CODE REVIEW

File: drivers/chrome.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
+ChromeDriver
+===============
+
+ChromeDriver allows Mink to control Chrome without the overhead of Selenium.
+
+It communicates directly with chrome over HTTP and WebSockets, which allows it to work at least twice as fast as Chrome with Selenium.
+
+For chrome 59+ it supports headless mode, eliminating the need to install a display server, and the overhead that comes with it.
```
Comment (aik099) [inline drivers/chrome.rst]: Shouldn't `chrome` be `Chrome` instead?

File: guides/drivers.rst [hunk 2]
```diff
@@ -32,23 +33,23 @@ cannot evaluate JavaScript and Selenium2Driver cannot get the response status
...
-Status code access      Yes                No         Yes     No        No
-Mouse manipulation      No                 Yes        Yes     Yes       Yes
-Drag'n Drop             No                 Yes        No      Yes       Yes
-Keyboard actions        No                 Yes        Yes     Yes       Yes
-Element visibility      No                 Yes        No      Yes       Yes
-JS evaluation           No                 Yes        Yes     Yes       Yes
-Window resizing         No                 Yes        No      No        No
-Window maximizing       No                 Yes        No      Yes       No
-======================  =================  =========  ======  ========  ====
+======================  =================  =========  ======  ======  ========  ====
```
Comment (aik099) [inline guides/drivers.rst]: 1. Are you sure, that `ChromeDriver` has `Yes` everywhere?
2. Does that apply also to headless mode?

Comment (adorin) [reply guides/drivers.rst]: I'm sure.

It implements the entire CoreDriver API. The driver tests suite passes with and without --headless https://gitlab.com/DMore/chrome-mink-driver/pipelines/23431065

Headless mode doesn't really have limitations. It even supports screenshots.

**Comment by adorin** [comment]
@aik099 will you merge?

**Comment by aik099** [comment]
Sure.

**Comment by aik099** [comment]
Merged. Thanks @adorin .

**Pull request merged**